woocommerce/woocommerce-gateway-payfast
# Fix security lint issues

Related P2 : https://wooengineering.wordpress.com/2022/12/05/automated-security-testing-of-woocommerce-extensions/?replytocom=1290#respond 

Plugin security issues file: https://docs.google.com/spreadsheets/d/1OcLUNrz99Ca_inYmlM_QVAk8FoczsQ1bioE31s7BYFI/edit#gid=0 

---

### Description
Fix the `UnsafePrintingFunction `, `OutputNotEscaped`, `NonceVerification`, `InputNotSanitized`, and the formatting errors.

### Steps to Test
* Go to the staging site https://zapaymentstaging.mystagingwebsite.com/ 
* Generally smoke test the plugin and make sure there are no regressions


### Changelog Entry
> Dev - Resolved linting issues.
